src/configuration/routes/cache.md

```diff
@@ -25,56 +23,64 @@ https://{default}/:
 
 In this example, requests will be cached based on the URI, the `Accept` header, `Accept-Language` header, and `X-Language-Locale` header; Any response that lacks a `Cache-Control` header will be cached for 60 seconds; and the presence of any cookie in the response will disable caching of that response.
 
-## Cache Per Route
+### Cache Attributes
 
-If you need fine-grained caching, you can set up caching rules for
-several routes separately.
+#### `enabled`
 
-Here is an example:
+When set to `true`, enable the cache for this route. When set to `false`, disable the cache for this route.
 
-```yaml
-https://{default}/:
-  type: upstream
-  upstream: app:http
-  cache:
-    enabled: true
+#### `headers`
 
-https://{default}/foo/:
-  type: upstream
-  upstream: app:http
-  cache:
-    enabled: false
+The `headers` key defines on which values the cache key must depend.
 
-https://{default}/foo/bar/:
-  type: upstream
-  upstream: app:http
-  cache:
-    enabled: true
+For example, if the `headers` key is the following:
+
+```yaml
+cache:
+  enabled: true
+  headers: ["Accept"]
 ```
 
-With this configuration, the following routes are cached:
+Then Platform.sh will cache a different response for each value of the `Accept` HTTP request header.
 
--   `https://{default}/`
--   `https://{default}/foo/bar/`
--   `https://{default}/foo/bar/baz/`
+> **note**
+> The following request headers cannot be used as cache key:
+> `Accept-Encoding`, `Connection`, `Proxy-Authorization`, `TE`, `Upgrade`.
 
-And the following routes are **not** cached:
+#### `cookies`
 
--   `https://{default}/foo/`
--   `https://{default}/foo/baz/`
+The `cookies` key define on which values the cache key must depend.
+
+For example:
+
+```yaml
+cache:
+  enabled: true
+  cookies: ["foo"]
+```
+
+The cache key will depend on the value of the `foo` cookie in the request.
+
+A special case exists if the `cookies` key has the `["*"]` value: it means that any request with a cookie will bypass the cache. This is the default value.
 
 > **note**
-> Regular expressions in routes are **not** supported.
+> You can not use wildcards in the cookie name. Either use a precise cookie
+> name, or match all cookies with a `"*"`. `"SESS*"` or `"~SESS"` are currently
+> not valid values.
+
+#### `default_ttl`
 
-## Cache key
+If the response does not have a `Cache-Control` header, the `default_ttl` key is used to define the cache duration, in seconds. The default value is `0`.
 
-To decide how to cache a response, Platform.sh will build a cache key
-depending on several factors and store the response associated with this
-key. When a request comes with the same cache key, the cached response will
-be reused.
+If the application code returns a Cache-Control header, or if your `.platform.app.yaml` file is configured to set a cache lifetime for static assets, then this value is ignored in favor of the application container's headers.
```

Comment:
Also, the `default_ttl` only applies to non-static responses, I believe - all static assets will have a Cache-Control header anyway with a max age defaulting to 0 (the default for `expires` in the web config).

And "application container's headers" => "application's headers"